openapi/openapi.yaml

```diff
@@ -52,6 +52,28 @@ paths:
                 items:
                   $ref: "#/components/schemas/LanguageCode"
 
+  /metricdescriptions:
```

Comment:
Could you change this to `/metric-descriptions`?

Reply:
Yes, sure! Thank you for the reference btw.